### PR TITLE
Release/0.9

### PR DESCRIPTION
**Pull request by darinc**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,10 +23,22 @@ RUN pip install --upgrade pip
...
 RUN pip install -U requests[security]
 ADD . /rapidpro
-COPY docker.settings.prod /rapidpro/temba/settings.py
+COPY docker.settings /rapidpro/temba/settings.py
 
-RUN apt-get install -y curl
-RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
+
+RUN apt-get install -y xvfb
+#RUN apt-get install -y wkhtmltopdf
```
Comment (teehamaral) [inline Dockerfile]: I think that we can remove the commented lines

File: Dockerfile [hunk 2]
```diff
@@ -36,9 +48,9 @@ RUN python manage.py collectstatic --noinput
 
 RUN touch `echo $RANDOM`.txt
 
-RUN python manage.py compress --extension=.haml --force
+#RUN python manage.py compress --extension=.haml --force
```
Comment (teehamaral) [inline Dockerfile]: I think that we need this one line to compress haml files

**Comment by teehamaral** [comment]
@darinc could you review my comments, please? Thanks.

File: Dockerfile (hunk 2 above)
Comment (darinc) [reply Dockerfile]: I believe this failed unless Redis was present.  I will move this to the entrypoint.sh script and push these changes.

**Pull request merged**